Created new cloud billing page (#4056)
mattermost/docs: source/overview/mattermost-cloud-overview.rst

Mattermost Cloud Professional offers all enterprise-grade administration features, including:

 - **Identity Management:** Single Sign-on, SAML and AD/LDAP user and group provisioning, guest management, and enterprise mobility management.
 - **Governance:** Message and file retention policies, system and team permission policies, custom terms of service, channel moderation settings, and compliance exports.  

To get started with Mattermost Cloud Professional, visit https://mattermost.com/mattermost-cloud.

Mattermost Cloud Enterprise
---------------------------

Mattermost Cloud Enterprise offers the isolation, security, and control of self-hosted editions but without the burden of managing deployment, maintaining uptime, or applying upgrades. Mattermost Cloud Enterprise offers all chat and administration features, integrated DevOps workflows, provides a 99.0% financially-backed guaranteed uptime, and enterprise-grade support.
  

> Every Mattermost Cloud Enterprise instance is deployed in a private environment within an AWS VPC dedicated to a single customer. Within that VPC, all the required resources to run, monitor, and administer Mattermost are deployed in isolation. These resources include a dedicated RDS Aurora database cluster and a dedicated Kubernetes cluster, deployed across multiple availability zones and managed by Kubernetes experts. 

Reviewer: ```suggestion
Every Mattermost Cloud Enterprise instance is deployed in a private environment within an AWS VPC dedicated to a single customer. Within that VPC, all the required resources to run, monitor, and administer Mattermost are deployed in isolation. These resources include a dedicated RDS Aurora database cluster and a dedicated Kubernetes cluster, deployed across multiple Availability Zones and managed by Kubernetes experts.
```

> This intentional isolation and dedicated deployment allow Mattermost Cloud Enterprise to offer other privacy-centric features, such as VPC Peering, bring-your-own-domain and certificates, mutual TLS, access to monitoring tools, databases to run custom analysis, and more.

Reviewer: ```suggestion
This intentional isolation and dedicated deployment supports other privacy-centric features, such as VPC Peering, bring-your-own-domain and certificates, mutual TLS, access to monitoring tools, databases to run custom analysis, and more.
```

Future releases will include an enterprise network integration with VPC Peering, TLS mutual authentication, and additional data residency options.

To get started with Mattermost Cloud Enterprise, visit: https://mattermost.com/cloud to speak to a Mattermost Account Executive.

FAQ
---

How much does Mattermost Cloud Cost?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Mattermost Cloud Professional is free for up to 10 users. After ten users, it is $10.00/user/month as a pay-as-you-go subscription. Please contact us for over 1000 users.

Reviewer: ```suggestion
Mattermost Cloud Professional is free for up to 10 users. After 10 users, it is $10 USD per user, per month as a pay-as-you-go subscription. If you have over 1000 users please contact us.
```

Mattermost Cloud Enterprise is an annual subscription. Please contact us for a quote.    

How can customers trial Mattermost Cloud?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> We offer ten users free as a trial option in our Mattermost Cloud Professional edition.  Customers under 11 users can use Mattermost Cloud Professional free for as long as they need.  All features are provided in this free offering. 

Reviewer: ```suggestion
Mattermost Cloud Professional edition is free for up to ten users and all features are provided in this free offering.
```

> Mattermost Cloud Enterprise offers a discounted proof-of-concept for 60 days. The proof-of-concept program includes a dedicated customer engineer.  

Reviewer: ```suggestion
Mattermost Cloud Enterprise offers a discounted proof-of-concept for 60 days. The proof-of-concept program includes a dedicated customer engineer.
```

Reviewer: This sounds very interesting - do we have a link to more info around it?

Author: Draft in progress: https://docs.google.com/document/d/17J8fgeuG9JRTI26hBhFlxV69ZcV4odLgC6kCixux5NA/edit#

> When will we support other regions?

Reviewer: ```suggestion
When will support for other regions be available?
```

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Mattermost Cloud Enterprise will support data residency in early 2021. We appreciate feedback from our customers on regional support in both editions.  

Reviewer: ```suggestion
Mattermost Cloud Enterprise will support data residency in early 2021. We appreciate feedback from our customers on regional support for both editions.
```

> How is Mattermost Cloud secured?

Reviewer: We refer to Mattermost Cloud, but also Mattermost Cloud Enterprise. We don't reference Mattermost Cloud Professional though. Is Mattermost Cloud aka Mattermost Cloud Professional?

Author: Mattermost Cloud incorporates both. I specified each product specifically when the information was different. Open to suggestions or feedback on being more explicit on this.

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Mattermost Cloud has encryption at rest and in-transit. We are SOC2 Type1 certified and working towards SOC2 Type 2.

Reviewer: ```suggestion
Mattermost Cloud has encryption at rest and in-transit. We are SOC2 Type 1 certified and working towards SOC2 Type 2.
```

Mattermost Cloud Enterprise is deployed in a private environment within an AWS VPC dedicated to a single customer. This architecture provides the isolation and security of an on-premises deployment while ensuring consistent performance and uptime of your Mattermost workspace.

What are the alternatives, and why should customers choose Mattermost Cloud?
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Other SaaS offerings are closed-source and focus on a very broad use-case. Mattermost Cloud is built on our open-source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering.  Mattermost has deep integrations and workflows that help builders and operators be more effective. Mattermost Cloud Enterprise is deployed in a private environment within an AWS VPC dedicated to a single customer. Other SaaS offerings are multi-tenant, meaning customers share resources. 

Reviewer: ```suggestion
Other SaaS offerings are closed-source and focus on a very broad usecase. Mattermost Cloud is built on our open source core offering, allowing customers to view our source code and have a high degree of transparency into Mattermost development processes. Mattermost provides a DevOps-centric offering and has deep integrations and workflows that help builders and operators be more effective.

Mattermost Cloud Enterprise is deployed in a private environment within an AWS VPC dedicated to a single customer. Other SaaS offerings are multi-tenant, meaning customers share resources.
```
